metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1917,4 +1964,19 @@ class MetalsLanguageServer(
     })
   }
 
+  private object JsonString {
```

Comment:
Overall, I am not sure we need the unapplies here. You can just as easily match on JsonString.